lib/hooks.tsx

```diff
@@ -24,20 +24,37 @@ export function usePreviousIfDeeplyEqual<T>(val: T) {
 }
 
 /**
- * Calls method `setter` on `box` with `value`, iff `value` is set & deeply
+ * Calls method `method` on `box` with `value`, iff `value` is set & deeply
  * different from before (and the box is alive)
  */
-export function useSetter<
+export function useMethod<
   V,
   S extends string,
   T extends TalkObject & Record<S, (val: V) => any>,
->(box: T | undefined, value: V | undefined, setter: S) {
+>(box: T | undefined, value: V | undefined, method: S) {
   value = usePreviousIfDeeplyEqual(value);
   useEffect(() => {
     if (value !== undefined && box?.isAlive) {
-      box[setter](value);
+      box[method](value);
     }
-  }, [setter, box, value]);
+  }, [method, box, value]);
+}
+
+/**
+ * Like {@link useMethod}, except `args` is an array that gets spread into
+ * the method call
+ */
+export function useSpreadMethod<
+  V extends any[],
+  S extends string,
+  T extends TalkObject & Record<S, (...args: V) => any>,
+>(box: T | undefined, args: V | undefined, setter: S) {
+  args = usePreviousIfDeeplyEqual(args);
+  useEffect(() => {
+    if (args !== undefined && box?.isAlive) {
+      box[setter](...args);
+    }
+  }, [setter, box, args]);
```

Comment:
```suggestion
>(box: T | undefined, args: V | undefined, method: S) {
  args = usePreviousIfDeeplyEqual(args);
  useEffect(() => {
    if (args !== undefined && box?.isAlive) {
      box[method](...args);
    }
  }, [method, box, args]);
```

To have it match `useMethod`

Reply:
Right, renamed one but missed the other